### PR TITLE
add stack pack guidance into each LH audit guide

### LINKED ISSUE #3674: Incorporating Stack Packs into web.dev
cc: @housseindjirdeh @paulirish @egsweeny 

### What page(s) need to be updated?

All Lighthouse [Opportunities](https://web.dev/lighthouse-performance/#opportunities) and [Diagnostics](https://web.dev/lighthouse-performance/#diagnostics) pages in web.dev that have applicable stack pack recommendations.

### Why is this update needed?

Stack packs are Google approved, framework specific suggestions that appear in Lighthouse audits within the Opportunities, Diagnostics, and Passed Audits tab. However, they aren't mentioned in the relevant web.dev articles. This would make the web.dev resources more complete, provide a space for additional elaboration on framework specific recommendations, and surface stack pack recommendations/best practices in more places for users to discover.

### What's the deadline?

None, just want to start a discussion :)

### PR DESCRIPTION
**Pull request by paulirish**
Fixes #3674

Changes proposed in this pull request:

- Add each LH audit's stack pack content into the documentation.

Built via https://github.com/GoogleChrome/lighthouse/compare/updatewebdevwstackpack

### CODE REVIEW

**Comment by kaycebasques** [comment]
Hellooooo just noting that this is on my radar and I'll get to it next week (fingers crossed)

**Comment by paulirish** [comment]
not stale

File: src/site/content/en/lighthouse-performance/dom-size/index.md [hunk 1]
```diff
@@ -70,6 +70,18 @@ another approach for improving rendering performance is simplifying your CSS sel
 See Google's [Reduce the Scope and Complexity of Style Calculations](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations)
 for more information.
 
+## Stack-specific guidance
+
+If you use any of these CMS's, libraries or frameworks, consider the following suggestions:
+
+### Angular
+
+Consider virtual scrolling with the Component Dev Kit (CDK) if very large lists are being rendered. [Learn more](https://web.dev/virtualize-lists-with-angular-cdk/).
```
Comment (kaycebasques) [inline src/site/content/en/lighthouse-performance/dom-size/index.md:79]:
```suggestion
Consider [virtual scrolling](/virtualize-lists-with-angular-cdk/) with the
Component Dev Kit (CDK) if very large lists are being rendered.
```

**Comment by kaycebasques** [comment]
I need to make various edits to the writing (breaking lines at 100 characters, etc.) and I don't think I can push to your fork so I will close this and continue in a new branch.

**Pull request closed without merging**